modules/adqueryBidAdapter.js

```diff
@@ -248,6 +300,76 @@ function buildRequest(validBidRequests, bidderRequest) {
     pageUrl = bidderRequest.refererInfo.page || '';
   }
 
+  if (isVideo) {
+    const baseRequest = {
+      id: bid.bidId,
+      cur: ['USD'],
+      site: {
+        page: bidderRequest.refererInfo.page,
+        domain: bidderRequest.refererInfo.domain
+      },
+      tmax: bidderRequest.timeout,
+      // Add GDPR/CCPA consent if applicable
+      regs: bidderRequest.gdprConsent ? {
+        ext: {
+          gdpr: bidderRequest.gdprConsent.gdprApplies ? 1 : 0,
+          consent: bidderRequest.gdprConsent.consentString
+        }
+      } : {},
+      // === USER ===
+      user: {
+        id: bidderRequest.userId ? bidderRequest.userId.uid2?.id : undefined,
```

Comment:
`userId` is not present on `bidderRequest` but on `validBidRequest` in your case its `bid`